Fix bug #14406 - Add missing translates to home space form

auroreallibe opened the pull request:
The missing translation had no text by default, so it was empty. It was really blocking for english or deutch users.

auroreallibe committed:
Fix bug #14406 - Add missing translates to home space form

The missing translation had no text by default, so it was empty. It was really blocking for english or deutch users.

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -23,6 +27,14 @@
+            <value lang="en">Yes##No</value>
+        </parameter>
+        <parameter>
+            <name>cols</name>
```
What is this new parameter `cols`? And where is the `values` parameter in German?

mmoqui replied:
Missing again the German translation of `value` here

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -51,6 +71,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -79,6 +111,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -103,6 +147,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -167,6 +239,14 @@
         </parameter>
+        <parameter>
+            <name>values</name>
+            <value lang="en">List##Carrousel</value>
```
In English: carousel (a single r)

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -167,6 +239,14 @@
+        </parameter>
+        <parameter>
+            <name>values</name>
+            <value lang="de">List##Carrousel</value>
```
In English: carousel (a single r)

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -167,6 +239,14 @@
+        </parameter>
         <parameter>
             <name>default</name>
             <value lang="fr">carousel</value>
```
In French: carrousel (two r)

auroreallibe replied:
In this part, carousel is just the key. Not a label. So I let "carousel"

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -187,6 +267,14 @@
             <language>fr</language>
         </label>
+        <label>
+            <labelName>How many news ?</labelName>
```
In English: `Number of news` should be the more appropriate translation to `Nombre d'actualités`
Otherwise, in English, no non-breaking space between the sentence and (interrogation / exclamation / ... ) marks

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -223,6 +319,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -255,6 +363,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -291,6 +407,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -323,6 +447,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -351,6 +483,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes, about the entire platform##Yes, only on this subspace##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -611,6 +811,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Open in a new tab</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -703,6 +923,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Open in a new tab</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -795,6 +1035,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Open in a new tab</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -887,6 +1147,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Open in a new tab</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -979,6 +1259,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Open in a new tab</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui commented on aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml:
```diff
@@ -1099,6 +1403,10 @@
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
```
Where is the parameter `values` in German?

mmoqui reviewed (changes requested):
In the XML resources, missing de translation of some values. I commented out in `data.xml`, but the missing is also in the others XMLs.

auroreallibe committed:
Responding to @mmoqui requested changes on the previous PR

Responding to @mmoqui requested changes on the previous PR
In the XML resources, missing de translation of some values. I commented out in data.xml, but the missing is also in the others XMLs.

Just one point was ignored because you ask to change the key (not a label)  define by default (carousel) . There is a notion of lang but I don't understand why, because it is just to defined the key  used by default

This part :
> @@ -167,6 +239,14 @@
             <name>values</name>
             <value lang="fr">Liste##Carrousel</value>
         </parameter>
+        <parameter>
+            <name>values</name>
+            <value lang="en">List##Carrousel</value>
+        </parameter>
+        <parameter>
+            <name>values</name>
+            <value lang="de">List##Carrousel</value>
+        </parameter>
         <parameter>
             <name>default</name>
             <value lang="fr">carousel</value>

mmoqui reviewed (changes requested):
Missing again a de translation in data.xml
For some elements in German, the text/value is translated in German whereas some others are in English. Why not to translate all in German?

auroreallibe committed:
Perfectionnement German translation

With the missing  German translate...
And translation of all the Germans' parts which  were in English
